backend/app/models/wikipedia_article/__init__.py

```diff
@@ -0,0 +1 @@
+#
```

Comment:
You can take advantage of Python namespaces here:

```
from .article import Article as Article  # disable Ruff warning
from .category import Category as Category  # have to disable the Ruff warning about unused export here
from .external_link import ExternalLink as ExternalLink
```

Then the consumers of these models do:

```
from app.models import wikipedia

wikipedia.Article
wikipedia.Category
```

Reply:
Yes, I am using namespaces for the models now.